_posts/2018-08-21-OSS-by-Default-Docs.markdown

```diff
@@ -0,0 +1,153 @@
+---
+layout: epic
+title: "Open Source by Default: Docs"
+date: 2018-08-21
+author: [orta]
+categories: [potential, artsy, culture, docs]
+series: Open Source by Default
+---
+
+Artsy is growing up. We have thousands of subscriber galleries paying 3 to 4 figure monthly fees. As we're starting
+to see a real market-fit, and have started to mature our organization. For example, this year we introduced product
+managers into our 8 year old company to help us figure out how to build the right things. We actually started having
+spare headcount on Engineering again, for the first time _in years_.
+
+As a part of maturing, our team has really had to start considering how to make parts of our culture explicit
+instead of implicit. We have new, awesome folks wanting to understand why things are the way they are as well as
+folk who are raising up to new responsibilities only to find them completely un-documented.
+
+In-part, a consistent team for so long hasn't made it worth the time to document things everyone knows, but growth
+is very happy to disrupt that balance. Now we're shipping documentation updates weekly, to all sorts of places. In
+trying to write an awesome document, which I'll cover later, I looked at how we had consolidated our documentation
+over the last few years, and saw that we had fragmented due the tensions around wanting to write publicly.
+
+This post covers that tension, and how we came about to the new docs setup.
+
+<!-- more -->
+
+## Fragmentation
+
+Prior to today, we had 4 main repos for documentation:
+
+- 🔒 `artsy/potential` (2015) - Our onboarding repo, and general docs hub
+- [`artsy/mobile`][a_m] (2015) - The mobile team's repo
+- [`artsy/guides`][a_g] (2016) - A place for standards and guides
+- [`artsy/meta`][a_mt] (2017) - Externally-facing docs for non-Artsy folk
+
+Now we have two:
+
+- 🔒 `artsy/potential` - Support docs, and private documentation
+- [`artsy/README`][a_rm] - Documentation Hub, split into sections
+
+When potential was created, it was first created as an open repo with the focus of on-boarding information. Over
+time it's scope grew to cover more general team, repo and setup documentation. We ended up debating whether it
+should be a private repo instead though.
+
+The key arguments for closing it were:
+
+- There are sensitive things we want to document
+- We might want to document things that live in a private repo
```

Comment:
This is a little circular. Like, I know what you're trying to say, but someone else might get tripped up about "we wanted to close the repo so we could document things in a closed repo." The reason why we wanted to document things in a closed repo was so that we could collaborate on them privately, which is a bit different from documenting sensitive things. I think.

Reply:
I think my writing was hard here, here's what I'm gonna replace it with:

> - We can to write about info which lives in other private repos with full context

It's like, there are other repos which are private that we want to talk about - so having a private place to talk about them means everyone knows they exist and can read through